examples/apm-sample/README.md

```diff
@@ -0,0 +1,23 @@
+# Raygun Crash Reporting + APM Sample
+
+This is a sample application to show how to use Raygun4Node and Raygun-APM together.
+
+This example uses the local `raygun4node` package in the project root directory by simply pointing to the root directory as a dependency in package.json:
+
+```
+"raygun": "file:../..",
+```
+
+## Run the sample
+
+First, install the `raygun4node` package.
+
+To do so, navigate to the project root directory, then:
+
+    npm install
```

Comment:
A customer yesterday was asking if we support `pnpm` instead. Investigate and please add to the docs as an alternative

Reply:
Let's make this an issue to resolved outside of this release